platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationContextImpl.java

```diff
@@ -196,6 +196,47 @@ public boolean cleanDirectory(Path path) {
     return true;
   }
 
+  /**
+   * Temporary version of cleanDirectory() that doesn't delete any directories but only files found.
```

Comment:
We should link the ticket number in this javadoc so this isn't lost.  Is there a ticket to remove this temporary change?  
https://codice.atlassian.net/browse/DDF-3383

Reply:
It is the same ticket number.